clarify how multiple layers of .attrs work (#720)
styled-components/styled-components-website: sections/basics/attaching-additional-props.md

```

As you can see, we get access to our newly created props in the interpolations, and the `type` attribute is passed down to the element.

### Overriding .attrs

> Notice that when wrapping styled components, `.attrs` are applied from the innermost styled component to the uppermost styled component.

Reviewer: ```suggestion
Notice that when wrapping styled components, `.attrs` are applied from the innermost styled component to the outermost styled component.
```

> This allows each level to **override** lower levels of `.attrs`, similarly to how css properties override css properties of inner components.

Reviewer: ```suggestion
This allows each wrapper to **override** nested uses of `.attrs`, similarly to how css properties defined later in a stylesheet override previous declarations.
```

> Here `Input`'s `.attrs` are applied first, and then `PasswordInput`'s `.attrs`:

Reviewer: ```suggestion
`Input`'s `.attrs` are applied first, and then `PasswordInput`'s `.attrs`:
```
